spec_parser/utils.py

```diff
@@ -162,6 +163,123 @@ def gen_rdf(self) -> None:
         with safe_open(fname, "w") as f:
             f.write(g.serialize(format="turtle"))
 
+    def gen_schema(self) -> None:
+        schema = {
+            "$schema": "https://spdx.org/spdx3/json_schema",  # TODO: insert correct schema and id URIs
```

Comment:
You should use the same version as the current schema - there was feedback/issues on using the most relevant version.